manifests/helm-profiles/openshift-ambient.yaml

```diff
@@ -15,10 +15,7 @@ cni:
   chained: false
   cniConfFileName: "istio-cni.conf"
   logLevel: info
```

Comment:
I think you need to change `manifests/profiles` && `manifests/helm-profiles` ATM?

Doing only this set will just change things for `helm install` but not for `istioctl install`

Reply:
the operator ones set the helm profile in them now